docs/csharp/whats-new/csharp-7-1.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in C# 7.1
+description: An overview of new features in C# 7.1.
+keywords: C# language design, 7.1, Visual Studio 2017, 
+author: billwagner
+ms.author: wiwagn
+ms.date: 08/16/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# What's new in C# 7.1
+
+C# 7.1 is the first point release to the C# Language. This release is the
+first step toward the next major release of the langauge. This increased
+cadence enables you to use the new features sooner, ideally when
+each features is ready. This release also adds the ability to configure
+the compiler to match a specified version of the language. This feature
+separates the decision to upgrade tools from the decision to upgrade language
+versions.
```

Comment:
Not sure if you're interested in feedback on the actually copy, but the last four sentences in this paragraph all start with "This", which reads a bit awkwardly.